### DISCUSSION TITLE
Mobile version?

### DISCUSSION THREAD

**Discussion by BigHonkinDaddy** [powertab/powertabeditor#464] [Ideas]
I would find it hard to compose something new on a mobile device.  However, I have grown accustomed to playing along with PowerTab's Midi player rather than just my metronome.  But the advantage the metronome has it is an app on my phone.  I'd love to be able to export the midi player (not just the audio, but also the animated tab play-back).  I may resort to a video capture of the midi playing on my pc and then playing the video on my phone, if no other option.

**Comment by cameronwhite** [comment]
This is kind of related to #149 - definitely a large project :) but I think limiting it to playback rather than editing is likely a much better approach to start with
